HISTORY.md

```diff
@@ -1,3 +1,24 @@
+## [15.1.2](https://github.com/theforeman/puppet-foreman/tree/15.1.2) (2020-11-11)
```

Comment:
I'd stop doing this. In stable branches we typically write them by hand anyway and this only creates noise in the git log. They make reviews harder.

Reply:
Done.